src/result_gen_with_api.py

```diff
@@ -278,13 +281,13 @@ def run_retroactive_flow():
                 rgb_artifacts)
         else:
             workflow_matched = False
-            print("Workflow id does not match with any of the id of registered Workflow")
+            logger.info("Workflow id does not match with any of the id of registered Workflow")
 
         if workflow_matched:
             try:
                 flow.run_flow()
-            except Exception as e:
-                print(e)
+            except Exception:
+                logger.exception("Error in Run Flow")
```

Comment:
we can also add the flow which had exception in this.

Reply:
logger.exception will log the full trace of the error along with the msg. So it will be clear that in which flow error has occured